internal/api/export/worker.go

```diff
@@ -114,6 +116,11 @@ func (s *Server) exportBatch(ctx context.Context, eb *model.ExportBatch) error {
 		logger.Infof("No records for export batch %d", eb.BatchID)
 	}
 
+	exposures, err = ensureMinNumExposures(exposures, eb.Region, s.config.MinRecords)
```

Comment:
Is it desirable to prevent inference that the export has been padded? We could add some jitter into the min length arg.

Reply:
good idea - done.